src/pip/_internal/vcs/bazaar.py

```diff
@@ -44,12 +44,12 @@ def fetch_new(
             display_path(dest),
         )
         if verbosity <= 0:
-            flag = "--quiet"
+            flags: Tuple[str, ...] = ("--quiet",)
```

Comment:
Why not use lists? It'd avoid all the annotations and be arguably more semantically correct.

Reply:
I do agree with that list would be more suitable, but I am trying to be consistent with flags management [here](https://github.com/pypa/pip/blob/c247ddce123513198858d4589c5ee198dcc6ba40/src/pip/_internal/vcs/git.py#L266-L271) and [here](https://github.com/pypa/pip/blob/c247ddce123513198858d4589c5ee198dcc6ba40/src/pip/_internal/vcs/mercurial.py#L46-L53). To be fair I don't know why tuples were used there.

Reply:
Some people use tuple as immutable list (which is wrong IMO but whatever). Anyway, this variable is internal to the function anyway, so there’s no need to match other code.